inc/sanitize/namespace.php

```diff
@@ -815,3 +817,26 @@ function maybe_safer_unserialize( $original ) {
 	}
 	return $original;
 }
+
+
+/**
+ * Sanitize an string for undesired XSS attacks allowing HTML but removing malicious code
+ * @param $value
+ * @param bool $allow_html
+ * @return string
+ */
+function sanitize_string( $value, $allow_html = false ) {
+
+	if ( $allow_html ) {
+
+		return HtmLawed::filter(
+			pb_decode( $value ), [
+				'safe' => 1,
+			]
+		);
+
+	}
+
+	return strip_tags( pb_decode( $value ) );
+
```

Comment:
Oscar, those 13 lines could be replaced by:
`return $allow_html ? HtmLawed::filter( pb_decode( $value ), [ 'safe' => 1 ] ) : strip_tags( pb_decode( $value ) );`

It is not mandatory, it is just a suggestion. The current implementation is cleaner that one-line solution in terms of readability but it still concentrate the single action (2 cases) simple to read.

If you decide to keep it in multiple lines, you can reduce it by deleting the empty lines to keep consistence between other functions in the same file.

Reply:
I sent the improvement, I think for this case the ternary operator is fine 👍 thanks